spec/jobs/delete_pii_from_old_claims_job_spec.rb

```diff
@@ -0,0 +1,19 @@
+require "rails_helper"
+
+RSpec.describe DeletePiiFromOldClaimsJob do
+  describe "#perform" do
+    let(:four_hours_ago) { 4.hours.ago }
+    let(:over_two_months_ago) { 2.months.ago - 1.day }
+
+    it "deletes the PII from eligible claims" do
+      rejected_claim = create(:claim, :rejected, updated_at: over_two_months_ago)
```

Comment:
It might be clearer to create a claim, and then use `travel_to` to go over two months into the future, rather than mutating the `updated_at` variable like that

Reply:
I prefer to set an explicit `updated_at` as it is less cognitive load than than wrapping multiple lines inside time travel blocks.